front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/RenderPopup.tsx

```diff
@@ -51,23 +53,50 @@ function RenderPopup() {
           <button
             className="btn btn-sm btn-success"
             type="button"
-            onClick={() => setPointIti('start', properties)}
+            onClick={() =>
+              setPointIti(
+                'start',
+                properties,
+                updateOrigin,
+                updateDestination,
+                addVias,
+                updateFeatureInfoClick
+              )
```

Comment:
Instantiate a const updateActions and pass it directly to setPointIti:
```suggestion
              setPointIti(
                'start',
                properties,
                updateActions
              )
```

Reply:
I wonder if it's not better to define `setPointIti` inside `RenderPopup` because if I apply your suggestion, the function become very short

Reply:
Done